paddle/parameter/ParameterUpdaterHook.cpp

```diff
@@ -29,40 +31,21 @@ namespace paddle {
 
 /**
  * The static pruning hook
- *
- * Static means user load a mask map before training started. This map will
- * define which link/weight between neural is disabled.
+ * Static means user specific a sparsity_ratio before training start, and the
+ * network will prune the parameters based on the sparsity_ratio. More deatils
```

Comment:
typo: specific -> specify, start -> started
More deatils can see -> More details can be found